.storybook/Divider.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import { Divider } from '../src'
+
+const description = '<hr /> with settings for padding, margin, width, and borderColor'
+
+storiesOf('Divider', module)
+  .add('Divider component', withInfo({
+    text: description,
+    inline: true
+  })(() => (
+    <Divider borderColor='lighterGray' />
+  )))
+  .add('Padding', () => (
+    <Divider p={3} borderColor='blue' />
+  ))
+  .add('Margin', () => (
+    <Divider m={3} borderColor='blue' />
+  ))
+  .add('Width', () => (
+    <Divider
+      p={3}
+      width={1/2}
+      borderColor='lighterGray' />
+  ))
+  .add('Pixel Width', () => (
+    <Divider
+      p={3}
+      width={256}
+      borderColor='lighterGray' />
+  ))
+  .add('VW Width', () => (
+    <Divider
+      p={3}
+      width='50vw'
+      borderColor='lighterGray' />
+  ))
+  .add('Directional Padding', () => (
```

Comment:
Maybe dumb question: is there a good use-case for padding on an `<hr>`? 🤔

Reply:
probably not, I think it's safe to nix the padding ability